mysql/metadata.csv

```diff
@@ -41,6 +41,7 @@ mysql.performance.slow_queries,gauge,,query,second,The rate of slow queries.,-1,
 mysql.performance.table_locks_waited,gauge,,,,The total number of times that a request for a table lock could not be granted immediately and a wait was needed.,-1,mysql,table locks waited,
 mysql.performance.table_locks_waited.rate,gauge,,,,"The rate of times that a request for a table lock could not be granted immediately and a wait was needed.",0,mysql,table locks waited rate,
 mysql.performance.threads_connected,gauge,,connection,,The number of currently open connections.,0,mysql,threads connected,cpu
+mysql.performance.users_connected,gauge,,connection,,The number of currently open user connections.,0,mysql,users connected,cpu
```

Comment:
`users_connected` is an innacurate name. This is more like `user_connections` since the count is a count of all rows in the threads view, not really a count of users. 

`COUNT(DISTINCT processlist_user) AS connections` would be a count of users (but I find this less useful).

Reply:
How is this metric different from `mysql.performance.threads_connected` and why are both needed?